### ISSUE TITLE
Generate service error with JS default config

### ISSUE THREAD

**Issue by onomated** [feathersjs/cli#190]
### Steps to reproduce
*  Replace `config/default.json` with `config/default.js`.  Valid configuration for [node config](https://github.com/lorenwest/node-config/wiki/Configuration-Files)
*  `feathers generate service`.  Follow prompts and get presented an error

### Expected behavior
Should generate a feathers service

### Actual behavior
Get the error log below:
```
$ feathers generate service
? What kind of service is it? A custom service
? What is the name of the service? mailer
? Which path should the service be registered on? /mailer
events.js:186
      throw er; // Unhandled 'error' event
      ^

TypeError: Cannot read property 'ts' of undefined
    at ServiceGenerator.writing (/.../.nodenv/versions/12.10.0/lib/node_modules/@feathersjs/cli/node_modules/generator-feathers/generators/service/index.js:167:18)
    at Object.<anonymous> (/.../.nodenv/versions/12.10.0/lib/node_modules/@feathersjs/cli/node_modules/yeoman-generator/lib/index.js:427:27)
    at /.../.nodenv/versions/12.10.0/lib/node_modules/@feathersjs/cli/node_modules/run-async/index.js:25:25
    at new Promise (<anonymous>)
    at /.../.nodenv/versions/12.10.0/lib/node_modules/@feathersjs/cli/node_modules/run-async/index.js:24:19
    at /.../.nodenv/versions/12.10.0/lib/node_modules/@feathersjs/cli/node_modules/yeoman-generator/lib/index.js:428:13
    at processImmediate (internal/timers.js:439:21)
Emitted 'error' event on ServiceGenerator instance at:
    at Immediate.<anonymous> (/.../.nodenv/versions/12.10.0/lib/node_modules/@feathersjs/cli/node_modules/yeoman-generator/lib/index.js:436:22)
```

**Comment by onomated** [comment]
Just realized.  This is the same issue mentioned here:  https://github.com/feathersjs/generator-feathers/issues/470

**Comment by daffl** [comment]
This has been fixed in `@feathersjs/cli@4.2.0`. If you are using TypeScript, make sure you add `"types": "lib/"` in your `package.json` for this to work.
